File: P1S3_JAVA/src/Cliente.java
public class Cliente {
    Formulario formulario;
    GestorFiltros gestorFiltros;

    public Cliente(GestorFiltros g){
        gestorFiltros = g;
        formulario = new Formulario();
    }

    public void enviarFormulario(){
        gestorFiltros.setFormulario(formulario);
        gestorFiltros.procesarFormulario();
    }

    public void setFormulario(Formulario f){
        formulario = f;
    }
}

File: P1/P1S3/Cliente.cpp
//
// Created by ines on 8/4/21.
//

#include "Cliente.h"

Cliente::Cliente(GestorFiltros g){
    gestorFiltros = g;
}

void Cliente::setFormulario(Formulario f){
    formulario = f;
}

void Cliente::enviarFormulario() {
    gestorFiltros.setFormulario(formulario);
    gestorFiltros.procesarFormulario();
}